build/dependencies.props

```diff
@@ -6,6 +6,7 @@
     <InternalAspNetCoreSdkVersion>2.1.0-*</InternalAspNetCoreSdkVersion>
```

Comment:
File contains a `$(CoreFxVersion)` setting but that shouldn't change in this repo. The property is only used in Microsoft.Extensions.* projects targeting .NET Standard 1.3 or earlier.